tests/mir-opt/building/match/sort_candidates.rs

```diff
@@ -0,0 +1,32 @@
+// skip-filecheck
```

Comment:
Since you're looking for specific patterns, it may be good to use filecheck to ensure it doesn't get blessed away in some larger change